src/components/TheNavBar.vue

```diff
@@ -134,15 +194,27 @@
     data() {
       return {
         showNav: false,
+        showAboutModal: false,
+        appName: "",
+        appVersion: "",
       }
     },
+    mounted() {
+      this.appName = this.$appName
```

Comment:
Not sure this is very helpful: values in data part are meant to be watched and dynamic. This can be accessed directly.